### DISCUSSION TITLE
Magick not working and I cant figure out

### DISCUSSION THREAD

**Discussion by EliasMerabet** [ImageMagick/ImageMagick#2068] [Help]
Hello community, just newbie using this feature.

After install magick nuget package, I code a very simple example for testing it.

using ImageMagick;
var image = new ImageMagickObject.MagickImage(bytes);

and:
var size = new MagickGeometry(maxWidth, maxHeight);

Are not recognized as a correct command.

I know looks like a simple error for my own, but I cant figure out what is happening.
Please help me.

**Comment by dlemstra** [comment]
Not sure which NuGet package you are talking about? I suspect it is Magick.NET and I don't understand why you would use `ImageMagickObject.MagickImage`. You should use `MagickImage` instead.

**Comment by EliasMerabet** [comment]
My mistake... I have to use MagickImage. Definitely was my double mistake,
I got to run it... Thanks mate.
